model/event/task_event.go

```diff
@@ -39,6 +39,7 @@ type TaskEventData struct {
 	UserId    string `bson:"u_id,omitempty" json:"user_id,omitempty"`
 	Status    string `bson:"s,omitempty" json:"status,omitempty"`
 	JiraIssue string `bson:"jira,omitempty" json:"jira,omitempty"`
+	Aborted   bool   `bson:"aborted", json:"aborted"`
```

Comment:
It seems we should probably avoid adding more boolean options (what happens when we need to add another field? or state information to a notification.) Seems like we need to get the task when we're trying to send the notification, we should just make the notification for tasks non-loggable if the task was aborted.

Reply:
I think this is what we do already (from #2319) but it doesn't prevent notifications from being sent for aborted tasks because the field is cleared by MarkEnd before we send the notification.

My idea was to add a field to the event so even though it's cleared from the task we will know if it had been aborted when we logged the event. Could you explain more about why you prefer not to add this field?